build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1598,6 +2268,10 @@ message CacheCapabilities {
 
   // Whether absolute symlink targets are supported.
   SymlinkAbsolutePathStrategy.Value symlink_absolute_path_strategy = 5;
+
+  // Support for two-phase caching for actions with possibly under-specified
+  // inputs.
+  TwoPhaseCacheSupport.Value two_phase_cache_support = 6;
```

Comment:
I am not sure a boolean would not be a better choice, ditto with the others.

Reply:
Changed to bool for ConsoleOut as part of #173 (and copied back here). Changed these to bool as well.